Mathlib/Data/Fintype/Basic.lean

```diff
@@ -114,6 +114,14 @@ theorem univ_eq_empty_iff : (univ : Finset α) = ∅ ↔ IsEmpty α := by
   rw [← not_nonempty_iff, ← univ_nonempty_iff, not_nonempty_iff_eq_empty]
 #align finset.univ_eq_empty_iff Finset.univ_eq_empty_iff
 
+theorem univ_nontrivial_iff [Fintype α] :
```

Comment:
I think the `[Fintype α]` assumption is already present at this point of the file.